plate_mapper/plate_mapper.py

```diff
@@ -97,20 +101,50 @@ def plate_mapper(input_f, barseq_f, output_f, names_f=None):
             barseqs.append(line.split('\t'))
     barseq_f.close()
     print('  Done.')
+
+    # Read special sample definitions
+    specs = {}
+    if special_f:
+        print('Reading special sample definitions...')
+        next(special_f)  # skip header line
```

Comment:
I'd normally agree with @wasade on this - there is a bit of redundant logic that is already implemented and tested in pandas.

However, given usability requirements and the tight timing, I am in favor of keeping this the way it is.